bin/check-process.rb

```diff
@@ -261,7 +270,10 @@ def run
     procs.reject! { |p| p[:command] =~ /#{config[:exclude_pat]}/ } if config[:exclude_pat]
     procs.reject! { |p| p[:command] !~ /#{config[:cmd_pat]}/ } if config[:cmd_pat]
     procs.select! { |p| p[:vsz].to_f > config[:vsz] } if config[:vsz]
+    # Ensure RSS is over this value
     procs.select! { |p| p[:rss].to_f > config[:rss] } if config[:rss]
+    # Ensure RSS is under this value
```

Comment:
I can clarify (or remove entirely) the comments I added there, but I'm not sure they are incorrect. Looking at both RSS-related selects:

```ruby
# Ensure RSS is over this value
procs.select! { |p| p[:rss].to_f > config[:rss] } if config[:rss]
# Ensure RSS is under this value
procs.select! { |p| p[:over_rss].to_f < config[:over_rss] } if config[:over_rss]
```
The first (and original) selects only those processes (`p`) whose reported RSS (`:rss`) is _greater than_ the supplied value in `config` (thus my comment of `# Ensure RSS is over this value`).

The second selects only those processes whose reported RSS is _less than_ the supplied value in the `config` (thus my comment of `# Ensure RSS is under this value`). Albeit, the use of "under" and the config key of `:over_rss` is not super clear, my intention on the new `-R` is to alert when the RSS value goes _over_ the value you specify on the command line, so I think it still fits.